PluginsAndFeatures/azure-toolkit-for-intellij/build.gradle

```diff
@@ -48,14 +48,6 @@ findbugs {
     ignoreFailures = true
 }
 
-intellij {
```

Comment:
Yes, we are defining `intellij` configuration for all projects later in the .gradle. I've cleared up this script a bit more now. The original changes came from upstream. Since we have split the plugin - this build.gradle contains a very general config for all projects. Moved it to java build.gradle now.